### PR TITLE
Web Accessible Resources for Manifest V3

### PR DESCRIPTION
**Pull request by solomonkinard**
Add public announcement about Web Accessible Resources.

### CODE REVIEW

**Comment by awfuchs** [comment]
> Deploy preview for _developer-chrome-com_ failed.
> 
> Built with commit [210dfa2](https://github.com/GoogleChrome/developer.chrome.com/commit/210dfa290f7909391f7759be169d5b3a33a4a6e4)
> 
> https://app.netlify.com/sites/developer-chrome-com/deploys/607d402beb5bcd000794ac06

Any idea what this preview failure is about?

File: site/en/blog/new-in-extensions-1/index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+layout: 'layouts/blog-post.njk'
+title: An example blog post
```
Comment (samthor) [inline site/en/blog/new-in-extensions-1/index.md:3]: missing title

File: site/en/blog/new-in-extensions-1/index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+## Summary
+
+Web Accessible Resources for Manifest is here! Now `manifest.json` supports permission definitions. Developers can restrict resources based on the requesting site origin or extension id.
+
+## Examples
+
+`manifest.json`
+
+Wildcard site:
+```
```
Comment (samthor) [inline site/en/blog/new-in-extensions-1/index.md:30]: Can you change all of these to "```json"?

File: site/en/blog/new-in-extensions-1/index.md (hunk 1 above)
Comment (solomonkinard) [reply site/en/blog/new-in-extensions-1/index.md:3]: Good catch.

File: site/en/blog/new-in-extensions-1/index.md (hunk 2 above)
Comment (solomonkinard) [reply site/en/blog/new-in-extensions-1/index.md:30]: SG

File: site/en/blog/new-in-extensions-1/index.md (hunk 1 above)
Comment (awfuchs) [reply site/en/blog/new-in-extensions-1/index.md]: Still looking for the preview. Does something need to happen to kick Netlify back to life?

Comment (samthor) [reply site/en/blog/new-in-extensions-1/index.md]: Netlify is reporting that it's deployed at the last commit, so you should be good?

**Review by samthor — COMMENTED** [review]
This PR looks fine but it also seems to have brought in a bunch of changes to `SharedArrayBuffer`, which are unrelated, and are also in your other PR. (And it renames "simeonv" to "dotproto").

Do you want to make these changes? Are they unintentionally being included?

**Comment by dotproto** [comment]
The weird history in this branch is my fault. I've created a new PR with a clean history at #698. I'm going to close out this PR.

**Pull request closed without merging**